### ISSUE TITLE
Can't run demo

### ISSUE THREAD

**Issue by CaptainJeff** [ParallelLLC/algorithmic_trading#22]
This project looks great but when i'm trying ot test it out i'm getting this error. I don't see any documentation about where to fetch/set that data

============================================================
🤖 ALGORITHMIC TRADING SYSTEM WITH FINRL & ALPACA
============================================================

📊 Data Source: csv
📈 Trading Symbol: AAPL
💰 Capital: $100,000
⏱️  Timeframe: 1m
🔧 Broker API: paper
🧠 FinRL Algorithm: PPO
📚 Learning Rate: 0.0003
🎯 Training Steps: 100,000

============================================================

📥 DATA INGESTION DEMO
------------------------------
Loading data from source: csv
ERROR:agentic_ai_system.data_ingestion:CSV file not found: data/market_data.csv
❌ Failed to load data
❌ Cannot proceed without data
